README.md

```diff
@@ -49,6 +49,9 @@ If you want `ember server` to start the Hoodie Server for you, you have to confi
 // config/environment.js
 if (environment === 'development') {
   ENV.hoodie = {
+    client: {
+      url: 'http://127.0.0.1:4201'
+    },
```

Comment:
yes, I agree, I’d leave it as just `client: {}` so people see that it can be configured?